core/node/libp2p/rcmgr_logging.go

```diff
@@ -47,11 +48,19 @@ func (n *loggingResourceManager) start(ctx context.Context) {
 			case <-ticker.C:
 				n.mut.Lock()
 				errs := n.limitExceededErrs
-				n.limitExceededErrs = 0
-				n.mut.Unlock()
-				if errs != 0 {
-					n.logger.Warnf("Resource limits were exceeded %d times, consider inspecting logs and raising the resource manager limits.", errs)
+				n.limitExceededErrs = make(map[string]int)
+
+				for e, count := range errs {
+					n.previousErrors = true
+					n.logger.Errorf("Resource limits were exceeded %d times with error %q. Consider inspecting logs and raising the resource manager limits.", count, e)
```

Comment:
Can we give a better hint of what logs to look at and what things to grep for?

Can we add a link for someone to learn about raising limits?  I assume we want to link to https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr